lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -226,19 +227,30 @@
 .forum-nav-thread-link {
   display: block;
   padding: ($baseline/4) ($baseline/2);
+  transition: none;
 
-  &.is-active {
-    background-color: $forum-color-background;
-    color: $base-font-color;
+  &.is-active,
+  &:hover {
+    color: $forum-color-background;
+    background-color: $forum-color-hover-thread;
```

Comment:
`$forum-color-active-thread` is already a thing (it's white in v1 and blue in v2??) so I'll make a `forum-color-reading-thread` I think.